src/PageContent/Settings/Settings.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import './StudentCoordinatorSettings.css';
+import OrganizationDetails from './OrganizationDetails';
+import AccountManager from './AccountManager';
+import { AccountType } from '../../Enums';
+import MemberAccount from './MemberAccount';
+import PersonalAccount from './PersonalAccount';
+import './SCSettings0.css';
+import './SCSettings1.css';
+
+class Settings extends Component {
+
+    render() {
+        let org = this.props.account;
+        if(org.accountType === AccountType.DONATING_AGENCY_MEMBER)
+            org = this.props.donatingAgency;
+        return (
+            <div>
+                {this.props.account ?
+
+                    <div> 
+                        <div className="container">
+                            <OrganizationDetails
+                                accountType={this.props.account}
+                                account={org}
```

Comment:
it's really confusing to have `accountType` and `account` the way you're using it. Maybe have 
```
daAccount={this.props.account}
org={org}
```
instead? so inside `OrganizationDetails` it's clear that one is `org` and the other one is just purely used for DA cases.

Reply:
Fixed.